scripts/ci/job_hooks/before_e2e.sh

```diff
@@ -19,6 +19,9 @@ FROM=$1;
 TO=$2;
 PARAMS=$3;
 
+rm -rf node_modules/
```

Comment:
you can run `npm ci` that does exactly that, removing node modules and running install